lib/plugin/allure.js

```diff
@@ -215,7 +215,13 @@ module.exports = (config) => {
     }
 
     err.message = err.message.replace(ansiRegExp(), '');
-    reporter.endCase('failed', err);
+    if (reporter.getCurrentTest()) {
+      reporter.endCase('failed', err);
+    } else if (err) {
+      // this means before suite failed, we should report this.
+      reporter.startCase('BeforeSuite failed');
```

Comment:
Can we get suite name and pass it for this method?

Reply:
resolved: https://github.com/Codeception/CodeceptJS/pull/2248/commits/b72a702897e37e935cf9cc3eed0a03e548975895